docs/src/main/asciidoc/blaze-persistence.adoc

```diff
@@ -0,0 +1,199 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Using Blaze-Persistence
+
+include::./attributes.adoc[]
+:config-file: application.properties
+
+Blaze-Persistence is a query builder on top of JPA that provides support for advanced SQL features and Entity-Views for efficient DTOs.
```

Comment:
I think it would be a good idea to have a longer introduction explaining what it brings to the plate and the problems it's supposed to solve.